File: TipCalculator/JavaScript/TipCalculator.js
const prompt = require('prompt-sync')({sigint: true});

// Formats to US Dollar

let USDollar = new Intl.NumberFormat('en-US', {
  style: 'currency',
  currency: 'USD'
});

let cost = prompt('How much was your meal? - $');
let tip = prompt('What percentage would you like to tip? - ')

cost = Number(cost);
tip = Number(tip);

function tipCalc(x, y) {
  let tipPercent = y / 100;

  let tipAmount = x * tipPercent;

  let totalBill = x + tipAmount;

  tipAmount = USDollar.format(tipAmount)
  totalBill = USDollar.format(totalBill)

  console.log("Cost of meal: " + x + "\n");
  console.log("Tip percentage: " + y + "%\n");
  console.log("Tip amount: " + tipAmount + "\n");
  console.log("Total cost: " + totalBill + "\n");
}

tipCalc(cost, tip);

File: TipCalculator/Python/TipCalculator.py
cost = int(input("How much was your meal? - $"))
tip = int(input("What percentage would you like to tip? - "))

def tipCalc(x,y):
    tipPercent = y / 100

    tipAmount = x * tipPercent

    totalBill = x + tipAmount

    print("Cost of meal: " + "${:,.2f}".format(x) + ".\n")
    print("Tip percentage: " + str(y) + "%.\n")
    print("Tip amount: " + "${:,.2f}".format(tipAmount) + ".\n")
    print("Total cost: " + "${:,.2f}".format(totalBill) + ".\n")

tipCalc(cost, tip)
